docs/sources/mimir/configure/about-versioning/index.template

```diff
@@ -0,0 +1,57 @@
+---
+aliases:
+  - ../configuring/about-versioning/
+description: Learn about guarantees for this Grafana Mimir major release.
+menuTitle: Versioning
+title: About Grafana Mimir versioning
+weight: 50
+---
+
+# About Grafana Mimir versioning
+
+This topic describes our guarantees for this Grafana Mimir major release.
```

Comment:
```suggestion
aliases:
  - ../configuring/about-versioning/
description: Understand what major-release features are supported, unsupported, or experimental in Grafana Mimir.
menuTitle: Major-release features
title: Understand major-release features available in Mimir
weight: 50
---
# Understand major-release features available in Mimir
The features that follow are available in this major release of Mimir.
```

Reply:
I agree that the title isn't quite right, but I think that reflects that there's two groups of information on this page:

* Information about deprecated and experimental features, and our policy around them
* Information about upgrading and backwards compatibility

What if we split the page in two? Then we could have a page titled something like "deprecated and experimental features", and another along the lines of "upgrading and backwards compatibility". The latter could link back to the former.

What do you think @osg-grafana?